partcad/src/partcad/wrappers/wrapper_render_png.py

```
@@ -0,0 +1,71 @@
+# Licensed under Apache License, Version 2.0.
+#
+
+# This script is executed within a python runtime environment
+# (no need for a sandbox) to speed up parallel rendering and
+# to reduce Python dependencies on the host environment
+
+import os
+import sys
+import tempfile
+
+import build123d as b3d
```

Ruff (0.8.2)

`build123d` imported but unused

Remove unused import: `build123d`

(F401)